src/Bot.js

```diff
@@ -14,18 +17,28 @@ class Bot {
             this.channels.get(channel).updated = date;
         }
     }
+
     /**
      * Configure a new channel
      * @param  {String}   channel  Channel ID
      * @param  {Number}   delay    Time between bumps in seconds
      * @param  {Function} callback Callback to execute upon revival
+     * @param  {Array}    tags     Tags to considered relevant
      * @throws When refused
      */
-    configure(channel, delay, callback) {
+    configure(channel, delay, callback, tags = []) {
```

Comment:
Think the signature is getting big enough to warrant an object?

Reply:
Yea